database/seeds/UsersTableSeeder.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Str;
+
+class UsersTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        DB::table('users')->insert([
+            'userName' => Str::random(10),
+            'email' => Str::random(10).'@gmail.com',
```

Comment:
Why `Str::random`? Don't you want to know the username and email for logging in?

Reply:
Good point. Just had no idea what I'm doing with seeders so I followed a model I found in the docs.